### PR TITLE
[Edit] C++ Math Functions: `exp()`

### LINKED ISSUE #993: [Edit] C++ Math Functions: `exp()`
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting
Clarify unclear content and/or terms
Add new content (definitions, codeblocks, etc.)

### Description (optional)

Requesting edits for the [exp()](https://www.codecademy.com/resources/docs/cpp/math-functions/exp) term entry under the C++: Math Functions concept.

Under `cpp/concepts/math-functions/terms/exp/exp()`, please make the following edits:

- Revise the opening sentence to refer to the term "exp()" by name (and **bold** this term, as well)
- In the "Syntax" section:
  - Rename any and all parameters to something more informative and descriptive, if possible.
  - If there is no text underneath the \```pseudo snippet, please add 1-2 sentences that describe the parameters of the `exp()` function as well as the return value.
- Remove any phrasing that is instructive in tone, as opposed to informational. For example, something like "Use the `exp()` function to..." should be revised to something akin to "The following example uses the `exp()` function to..."
- If there is no Codebyte example, please add a new "Codebyte Example" section with 1-2 sentences previewing a \```codebyte/cpp code snippet.

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by borntofrappe**
### Description

Closes #993 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/exp/exp.md [hunk 1]
```diff
@@ -1,55 +1,56 @@
...
 
 ## Codebyte Example
 
-Use `exp()` to return the value of _e_ (*e*¹):
+The following example is runnable and returns the exponential function of `1`:
 
 ```codebyte/cpp
 #include <iostream>
-#include <cmath>
+#include <math.h>
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/exp/exp.md:48]: Not sure about this but isn't `math.h` more relevant to C programs as opposed to C++ programs? cc: @SSwiniarski

File: content/cpp/concepts/math-functions/terms/exp/exp.md [hunk 2]
```diff
@@ -1,55 +1,56 @@
...
 
-result = std::exp(3);
+int main() {
+  double x = 3;
+  double result;
+  result = exp(x);
+  std::cout << "The exp of " << x << " is " << result << "\n";
+  // Output: The exp of 3 is 20.0855
+}
 ```
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/exp/exp.md:41]:
```suggestion
}

This produces the following output:

```

And then add a new \```shell snippet featuring the output:

```shell
The exp of 3 is 20.0855
```

File: content/cpp/concepts/math-functions/terms/exp/exp.md [hunk 3]
```diff
@@ -1,55 +1,56 @@
...
-std::exp(n)
+ exp(x)
 ```
 
+The `x` parameter can be any value, positive, negative or zero.
+
 ## Example
 
-Use `exp()` to return the value of *e*³):
+The following example uses the `exp()` function to find the exponential function of `3`:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/exp/exp.md:27]:
```suggestion
The following example uses the `exp()` function to find the value of e<sup>3</sup>:
```

File: content/cpp/concepts/math-functions/terms/exp/exp.md [hunk 4]
```diff
@@ -1,55 +1,56 @@
...
   - 'learn-c-plus-plus'
   - 'paths/computer-science'
 ---
 
-`exp()` returns the base-_e_ exponential function of the argument, which is _e_ raised to the power of the argument.
-
-The number _e_, also known as Euler's number, is a mathematical constant approximately equal to 2.71828 and the base of the natural logarithm.
-
-The `cmath` library must be added to the top of the file with `#include <cmath>`.
+The **exp()** function returns the base-_e_ exponential function of the argument. This is _e_ raised to the power of the argument. This is the inverse of the [`log()`](https://www.codecademy.com/resources/docs/cpp/math-functions/log) function.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/exp/exp.md:15]:
```suggestion
The **exp()** function returns the base-_e_ exponential value of the argument (i.e., _e_ raised to the power of the argument). This is the inverse of the [`log()`](https://www.codecademy.com/resources/docs/cpp/math-functions/log) function.
```

File: content/cpp/concepts/math-functions/terms/exp/exp.md [hunk 5]
```diff
@@ -1,55 +1,56 @@
...
 ## Syntax
 
-Returns *e*ⁿ:
-
 ```cpp
-std::exp(n)
+ exp(x)
 ```
 
+The `x` parameter can be any value, positive, negative or zero.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/exp/exp.md:23]:
```suggestion
The `x` parameter is a `double` value that can be positive, negative, or zero.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@borntofrappe Thank you for your contribution to Docs! 😄 I just finished the first review for this PR and left you some suggestions/comments to look over. Tag me when you've finished pushing new changes or if you have any questions.

File: content/cpp/concepts/math-functions/terms/exp/exp.md (hunk 1 above)
Comment (borntofrappe) [reply content/cpp/concepts/math-functions/terms/exp/exp.md:48]: Apologies, I looked at the `log` function and it used `<math.h>`. I guess you could raise an issue to update the [log](https://www.codecademy.com/resources/docs/cpp/math-functions/log) entry as well.

File: content/cpp/concepts/math-functions/terms/exp/exp.md (hunk 3 above)
Comment (borntofrappe) [reply content/cpp/concepts/math-functions/terms/exp/exp.md:27]: I will use this suggestion for the section devoted to the Codebyte example.

```diff
-The following example is runnable and returns the exponential function of `1`:
+The following example is runnable and returns the value of e<sup>1</sup>:
```

File: content/cpp/concepts/math-functions/terms/exp/exp.md [hunk 6]
```diff
@@ -1,43 +1,52 @@
...
-The number _e_, also known as Euler's number, is a mathematical constant approximately equal to 2.71828 and the base of the natural logarithm.
-
-The `cmath` library must be added to the top of the file with `#include <cmath>`.
+The **exp()** function returns the base-_e_ exponential value of the argument (i.e., _e_ raised to the power of the argument). This is the inverse of the [`log()`](https://www.codecademy.com/resources/docs/cpp/math-functions/log) function.
 
 ## Syntax
 
-Returns *e*ⁿ:
-
 ```cpp
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/exp/exp.md:19]:
```suggestion
```pseudo
```

The \```pseudo code snippet is commonly used in the "Syntax" section of an entry.

**Review by Dusch4593 — APPROVED** [review]
@borntofrappe LGTM! Just one last suggestion from me but we are ready for the next review. 😄

**Comment by Dusch4593** [comment]
@SSwiniarski Awesome! Two approvals seems sufficient to merge this PR. 😄

@borntofrappe I'll let you know when the changes go live on the site. Thank you again for your contribution. 🚀

**Comment by Dusch4593** [comment]
The changes are now live: https://www.codecademy.com/resources/docs/cpp/math-functions/exp

However, @borntofrappe, I'm not seeing your profile appearing under the list of contributors for this entry 🤔:

Screen Shot 2022-10-05 at 12 16 55 PM

Do you know if your GitHub account is connected with your Codecademy? If we connect them, and then sign out and back in through GitHub, you should see it appear there.

**Comment by borntofrappe** [comment]
> Do you know if your GitHub account is connected with your Codecademy? If we connect them, and then sign out and back in through GitHub, you should see it appear there.

Should be connected now. Thanks for letting me contribute. 👍

**Comment by Dusch4593** [comment]
@borntofrappe Do you see your profile? https://www.codecademy.com/resources/docs/cpp/math-functions/exp

**Comment by borntofrappe** [comment]
@Dusch4593 not right now. Maybe it's a matter of time though.

**Comment by Dusch4593** [comment]
@borntofrappe There you are! 🚀 🚀 🚀 

Screen Shot 2022-10-06 at 9 08 39 AM

**Comment by borntofrappe** [comment]
> @borntofrappe There you are! 🚀 🚀 🚀

Thanks for the check up, and for accepting the pull request :) 
Let me know if you need more help 👍

**Pull request merged**